controllers/lvmcluster_controller.go

```diff
@@ -115,14 +118,12 @@ func (r *LVMClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	result, reconcileError := r.reconcile(ctx, lvmCluster)
 
 	statusError := r.updateLVMClusterStatus(ctx, lvmCluster)
-	if statusError != nil {
-		r.Log.Error(statusError, "failed to update VG Node status")
-	}
 
 	// Reconcile errors have higher priority than status update errors
 	if reconcileError != nil {
 		return result, reconcileError
-	} else if statusError != nil && errors.IsNotFound(statusError) {
+	} else if statusError != nil && !errors.IsNotFound(statusError) {
```

Comment:
Do we don't need to return errors in the case of `NotFound` scenarios now?

Reply:
I think this was originally a mistake (as Nitin pointed out).